Example/Sources/View Controllers/MessageSubviewContainerViewController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  MessageSubviewContainerViewController.swift
```

Comment:
This needs proper MIT License Header, copy that from `AppDelegate.swift` for example please :)